src/app.ts

```diff
@@ -55,6 +55,14 @@ const botManager = new BotManager();
 
 import ON_DEATH from 'death';
 
+// @ts-ignore
+// @TODO Fix this error.
+// Argument of type '(signal: any, err: any) => void'
+// is not assignable to parameter of type
+// '(signal: "SIGINT" | "SIGTERM" | "SIGQUIT" | "SIGHUP" | "uncaughtException") => void'.ts(2345)
+//
+// The parameter `err` does not exist in the typescript definion.
+// If `err` does exist, please open a PR against `@types/death`.
```

Comment:
https://github.com/jprichardson/node-death#example

Reply:
It is a thing so I guess we should make a PR

Reply:
I stand corrrected, he does some dynamic `arguments` tricks.
I'll just ts-ignore it, since not having types for the library at all is worse.

https://github.com/jprichardson/node-death/blob/master/lib/death.js#L11-L35

Reply:
Wait a second, I am not a javascript parser, but this seems weird.
```JS
ON_DEATH({ uncaughtException: true })(function(signal, err) { // bla bla bla
        ^                           ^
```

The (function(signal, err) {} is not passed as an argument.
It just floats right next to the call, since the `(` already closed.
Is this some new form of javascript that I am not aware of?